pyspecProcScripts/phasing.py

```diff
@@ -283,14 +342,21 @@ def filter_ranges(B, A):
         if show_shifted_residuals:
             d.set_plot_color(thiscolor)
         d.setaxis(direct, lambda x: x - test_shift).register_axis({direct: 0})
-        ph0 = zeroth_order_ph(select_pathway(d,
-            signal_pathway)[direct:0.0],
-            fl=zeroth_fl)
+        if signal_pathway is not None:
+            ph0 = zeroth_order_ph(select_pathway(d,
+                signal_pathway)[direct:0.0],
+                fl=zeroth_fl)
+        else:
+            ph0 = zeroth_order_ph(d[direct:0.0],fl = zeroth_fl)
```

Comment:
I think instead, just modify `select_pathway` so that it accepts `None` as an argument (and just returns the original data)